### ISSUE TITLE
What happened to the LIbreOffice plugin?

### ISSUE THREAD

**Issue by seatrout** [ccseer/Seer-plugins#6]
Seer used to display the content of libreoffice files. Now it has stopped doing so, and I can't find the relevant plugin. There is an old one somewhere on github which is marked as having been merged with the main branch ass the second pull request. But it only works up to v5, and LO is now on v7. And it has the wrong paths coded into it, so far as I can tell. ("Program Files x86" -- the current versions of LO are in plain "Program Files").

Changing the paths looks really simple, I mean so simple that I could have a hack at it. But I can't find the file to hack at, since it seems to have been dropped from the Seer repository altogether.

**Comment by ccseer** [comment]
The office file can be supported by OIT-1.1.0

**Comment by seatrout** [comment]
How do I get the preview then? I have the OIT-1.10 plugin installed and it does nothing with LO files.

**Comment by ccseer** [comment]
What is the extension?
write them into the field.
Make sure you have Seer-2.6.1
[image: image.png]

**Comment by seatrout** [comment]
I mean, the OIT 1.1.0 plugin ... Nor do any of the Libreoffice file formats (.odt, .ods, etc) show up when I type them into the search box. See screenshots. 
![2021-01-22 09_14_03-Settings - Seer](https://user-images.githubusercontent.com/5279653/105471199-61c1bf00-5c92-11eb-9b41-184b5ade999b.jpg)
![2021-01-22 09_13_39-Settings - Seer](https://user-images.githubusercontent.com/5279653/105471203-625a5580-5c92-11eb-8eb2-8a1f21a4a9bc.jpg)

**Comment by ccseer** [comment]
Double click OIT-1.1.0, then three input fields show up. After that, add
the office formats into that specific one


[image: image.png]

**Comment by seatrout** [comment]
I double clicked as you suggested, and typed ODT into the list of supported file extensions and pressed the tick. Saved the settings. Now it works! Thank you. Is there any reason not to have the OO/LO file extensions there by default? I'm sure it used to work without any fiddling before.

But, again, thank you for this helpful support.

**Comment by ccseer** [comment]
It is not that common as MS office.

**Comment by seatrout** [comment]
I agree it's not as common. But if the fix is as simple as adding a couple of file extensions to the default list, you could save some people a lot of confusion and some trouble!
